.github/workflows/test.yml

```diff
@@ -50,6 +50,7 @@ jobs:
         include:
           - os: ubuntu-latest
             wasmer-feature: wasmer_sys
+            dpki-feature: unstable-dpki
```

Comment:
Run test with DPKI enabled on Ubuntu. That safeguards against regressions and keeps potential of flakey tests at bay.

Reply:
Yes we want this, I think we probably don't want to run all the tests with every feature. I'd love if we had better tools for filtering tests but as we don't, maybe we should run the tests with all unstable features enabled as a single test run? Basically like we do now, so once we have all these PRs in we can gather these up and run them as a set?

Reply:
I'll merge this in as is and we can test if `--all-features` does not cause the segfault because of instrumentation?